duckduckgo/BrowserServicesKit
# add support for pixel based crash collection

Please review the release process for BrowserServicesKit [here](https://app.asana.com/0/1200194497630846/1200837094583426).

**Required**:

Task/Issue URL:  https://app.asana.com/0/1200194497630846/1203353338544857/f
iOS PR:  https://github.com/duckduckgo/iOS/pull/1381
macOS PR: n/a
What kind of version bump will this require?: Minor (new API only)

**Description**:

Uses MetricKit to gather crash info and report that back to the calling app.

**Steps to test this PR**:
1. Check out corresponding iOS branch (BSK is dropped into Xcode)
1. Launch the app *on a device* and then disconnect from debugger
1. Use the debug menu to crash the app using a fatal error
2. Re-launch the app, observe the crash pixel being fired




**OS Testing**:


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
